### DISCUSSION TITLE
The Klipper site and the Hungarian language

### DISCUSSION THREAD

**Discussion by ghost** [KlipperScreen/KlipperScreen#739] [Show and tell]
Hello! I am constantly improving the Hungarian language for the Klipper site via Weblate. Unfortunately, I don't see the language being updated. There are several things that are still in English, even though they were already translated 2-3 weeks ago. Here are one or two examples.
![Screenshot_20220930-194528](https://user-images.githubusercontent.com/45710979/193327362-c984cb50-06c3-40de-a2bf-8a3bd581a64a.jpg)
![Screenshot_20220930-194508](https://user-images.githubusercontent.com/45710979/193327365-b27ed1c4-ab49-47ee-87c7-ddd63e1065bf.jpg)

**Comment by alfrix** [comment]
you are referring to klipper documentation, and should open an issue on [the main repository](https://github.com/Klipper3d/klipper) or discuss it on [discourse](https://community.klipper3d.org/)

if there is an string not translated in the KlipperScreen interface please open an issue in this repository.

**Reply by ghost** [reply]
> you are referring to klipper documentation, and should open an issue on [the main repository](https://github.com/Klipper3d/klipper) or discuss it on [discourse](https://community.klipper3d.org/)
> 
> if there is an string not translated in the KlipperScreen interface please open an issue in this repository.

I don't even understand how it got here. The phone may have been jammed.
